modules/nextflow/src/main/groovy/nextflow/ast/NextflowDSLImpl.groovy

```diff
@@ -1254,7 +1254,7 @@ class NextflowDSLImpl implements ASTTransformation {
                 unit.addError( new SyntaxException("Identifier `$name` is reserved for internal use", node.lineNumber, node.columnNumber+8) )
                 return true
             }
-            if( name in functionNames || name in workflowNames || name in processNames ) {
+            if( /*name in functionNames ||*/ name in workflowNames || name in processNames ) {
```

Comment:
Not sure this is needed to remove it. It should prevent the use of reserved named for function if i'm not wrong

Reply:
if we want to have exactly one method (with default args) we can maintain but if we want to have the same method with different signatures is required to remove it from the check